.env

```diff
@@ -1,2 +1,2 @@
 DSBD_API_HOST="localhost"
-DSBD_API_PORT="3000"
\ No newline at end of file
+DSBD_API_PORT="3001"
```

Comment:
we should remove `.env` and add it to gitignore really rather than editting this. Doesn't have to happen in this PR if it's awkward but maybe give it a go

Reply:
removed `.env`